recipe/meta.yaml

```diff
@@ -29,7 +29,7 @@ requirements:
     - ninja
     - pkg-config
   host:
-    - c-blosc2
+    - c-blosc2 >=2.17
```

Comment:
This  isn’t the way to go unfortunately

Reply:
Can you please revert this in a new PR

bump then build number. And give me time to review? 

Basically this syntax in the run section stops the C dependency from staying in sync with the rest of the conda ecosystem.

Reply:
Ooops.  https://github.com/conda-forge/python-blosc2-feedstock/pull/28

Shout if you need something more on my side.  Thanks for taking care of this.